locallibrary/catalog/templates/catalog/book_detail.html

```diff
@@ -0,0 +1,26 @@
+{% extends "base_generic.html" %}
+
+{% block content %}
+    <h1>Title: {{ book.title }}</h1>
+
+    <p><strong>Author:</strong> <a href="">{{ book.author }}</a></p> <!-- author detail link not yet defined -->
+    <p><strong>Summary:</strong> {{ book.summary }}</p>
+    <p><strong>ISBN:</strong> {{ book.isbn }}</p>
+    <p><strong>Language:</strong> {{ book.language }}</p>
+    <p><strong>Genre:</strong> {{ book.genre.all|join:", " }}</p>
+
+    <div style="margin-left:20px;margin-top:20px">
+        <h4>Copies</h4>
+
+        {% for copy in book.bookinstance_set.all %}
+            <hr>
+            <p class="{% if copy.status == 'a' %}text-success{% elif copy.status == 'm'%}text-danger{% else %}text-warning{% endif %}">
+                {{ copy.get_status_display }}
+            </p>
+            {% if copy.status != 'a' %}
+                <p><strong>Due to be returned:</strong> {{ copy.due_back }}</p>
+            {% endif %}<p><strong>Imprint:</strong> {{ copy.imprint }}</p>
+            <p class="text-muted"><strong>Id:</strong> {{ copy.id }}</p>
+        {% endfor %}
+    </div>
+{% endblock %}
```

Comment:
end of file -> thêm dòng trống cuối file

Reply:
resolved